inst/EpiModel2.rmd

```diff
@@ -188,24 +199,28 @@ The following section will help EpiModel users update code written in EpiModel 1
 New `tergmLite` version of network initialization function:
```

Comment:
suggestions for line 185: copy-edits - note: suggest consistency in using `set` vs. set or `dat` vs. dat (leave it up to you for which you go with)

"assigned back" to what?

> Here we are pulling the arrival rate, `a.rate`, from the parameter list and querying the epidemic tracker list for the population size at the previous step. Note that the `set` functions do not modify `dat` in place but return an altered `dat` object that must be assigned back for the changes to be saved. For example, `dat <- set_attr(dat, "status")`.